config.toml.in

```diff
@@ -7,6 +7,8 @@
 # The storage section controls the configuration of the backend.
 [storage]
 type = "file"
+auto_backup = true
```

Comment:
Looks like you've added ` general` section [here] (https://github.com/leonklingele/pick/blob/61bc425e552cb72940d581a4eae9cc99254c1f10/config.toml.in#L12), so let's stick `auto_backup` in there, too.